integration/client/container_linux_test.go

```diff
@@ -1091,9 +1093,24 @@ func TestContainerKillInitPidHost(t *testing.T) {
 }
 
 func TestUserNamespaces(t *testing.T) {
-	t.Run("WritableRootFS", func(t *testing.T) { testUserNamespaces(t, false) })
+	var (
+		uidmaps        = []specs.LinuxIDMapping{{ContainerID: 0, HostID: 1000, Size: 65535}}
+		gidmaps        = []specs.LinuxIDMapping{{ContainerID: 0, HostID: 2000, Size: 65535}}
+		cmdExit        = withExitStatus(7)
+		cmdCheckSetuid = withProcessArgs("bash", "-c", "[ -u /usr/bin/passwd ] && exit 7")
```

Comment:
I'd add a comment to explain this 7. It's not obvious.

Reply:
7 was the value used in the existing tests. I simply reused it. I guess it's just a random value to differentiate from common exit code like 0 or 1.

Reply:
I know, but it is not so obvious now. I think a comment will help

Reply:
OK, I refactored that to table tests, which should be self explanatory. Does it look better now?

Reply:
Sorry the confusion, I was just saying to add a comment there. But yes, table test do look way nicer here, IMHO, so please keep them.

But can you add a comment on top of the lines that add the return 7, explaining that `testUserNamespaces()` uses that return code, etc.? Just a one-line comment should be enough, IMHO.

Reply:
Ah okey. updated.